### PR TITLE
Made new changes

### PR DESCRIPTION
**Pull request by khushaljobanputra**
I set up server to listen at port 3000
mongoDB connection set to port 27017 which is default port
Made many new changes.
I changes the way routes were written in the router. Will revert back if this way doesn't work well.
I created this branch just so that you guys can take a look at the changes I made and then merge it to the original once you guys feel it is not a bad update.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+//var favicon = require('serve-favicon');
 var logger = require('morgan');
 var cookieParser = require('cookie-parser');
 var bodyParser = require('body-parser');
 var mongoose = require('mongoose');
 var api = require('./routes/api');
 var app = express();
 
-var mongoDB = 'mongodb://127.0.0.1/my_database';
+var mongoDB = 'mongodb://localhost:27017';
```
Comment (shifatul-i) [inline app.js]: use 127.0.0.1 instead if localhost

Comment (khushaljobanputra) [reply app.js]: Okay will make that change

File: app.js [hunk 2]
```diff
@@ -50,4 +50,6 @@ app.use(function (err, req, res, next) {
     res.render('error');
 });
 
+app.listen(3000);
```
Comment (shifatul-i) [inline app.js]: remove this, we don't need this, we r using the /bin/www file this task

File: routes/api.js [hunk 3]
```diff
@@ -14,19 +14,38 @@ mongoose.Promise = promise;
...
+            });
+            news.save(function (err) {
+                if (err) return res.sendStatus(500)
+             })
+        }
+        //The else below returns an error if a non user is trying to add data to the database
+        else{
+            res.sendStatus(401);
+        }
+        res.sendStatus(201);
```
Comment (shifatul-i) [inline routes/api.js]: this will cause an error

File: app.js (hunk 2 above)
Comment (khushaljobanputra) [reply app.js]: I was just using it to check locally using the browser if the requests are being served. I will remove it once i finish writing up all the requests

File: routes/api.js [hunk 4]
```diff
@@ -14,19 +14,38 @@ mongoose.Promise = promise;
...
+    });
 
-router.get('/news/:id', function (req, res) {
-    res.send({news: []});
-}),
+router.route('/news/:id')
+    .get(function (req, res) {
+        res.send({news: []});
+    })
+    .post(function (req, res) {
```
Comment (shifatul-i) [inline routes/api.js:51]: we don't need post here, put and delete would be more sensible options here

File: routes/api.js (hunk 3 above)
Comment (khushaljobanputra) [reply routes/api.js]: Will remove the second res.sendStatus

File: app.js (hunk 2 above)
Comment (shifatul-i) [reply app.js]: and, run app using command 'npm start'. do not run the app.js file

File: routes/api.js (hunk 4 above)
Comment (khushaljobanputra) [reply routes/api.js:51]: Okay will make the change

File: app.js (hunk 2 above)
Comment (khushaljobanputra) [reply app.js]: Okay

**Comment by khushaljobanputra** [comment]
I'll make all the changes and push in an hour or two

**Pull request closed without merging**